### DISCUSSION TITLE
Hub54 - Possible ?

### DISCUSSION THREAD

**Discussion by OZ2RNH** [LAutour/ESP32-HUB75-MatrixPanel-I2S-DMA-icn2053#3] [Q&A]
Hi.

I was wondering if it would be possible to use this project for Hub54 panels, they are 26 Pins, and almost has the same pinout as the Hub75, just more RGB pins.

Hub75 is:
R1, G1, B1, R2, G2, B2.

And Hub 54 is:
R1, G1, B1, R2, G2, B2, R3, G3, B3, R4, G4, B4, R5, G5, B5.

I would be willing to donate a panel if it would help.

Regards
Ronni

**Comment by LAutour** [comment]
Need to think

**Comment by LAutour** [comment]
What is the screen size?

**Reply by OZ2RNH** [reply]
They are 40x40, P8 panels.

**Comment by LAutour** [comment]
Not ready yet but will be added in the new repository
https://github.com/LAutour/ESP32-HUB75-MatrixPanel-DMA-ICN2053

**Reply by OZ2RNH** [reply]
I am looking forward to trying it out. :) Thank you for looking into it.
